docs/scripts/gen-faq.js

```diff
@@ -41,9 +41,10 @@ import {Question, Faq} from '@theme/Faq'
 md = new Remarkable()
 faq.forEach((el) => {
   react_tags = el.tags.map((tag) => {
-    return tag + '_src-theme-'
+    return tag
```

Comment:
The whole `.map()` here is redundant as it does nothing

Reply:
wuut, i removed those.

Reply:
ahh yes you're right